api/controllers/Irc_EntryController.js

```diff
@@ -64,31 +67,36 @@ module.exports = {
 
   process_event: function (request_body) {
     if (request_body.operation === 'check in') {
-      return this.saveDetainee(request_body)
-        .then(this.saveEvent.bind(this, request_body));
+      return this.saveEvent(request_body);
     }
     throw new ValidationError('Unknown');
```

Comment:
we validate the payload with the json schema.
the only way you'll get here is if they've given a valid payload but an unhandled operation